news/2020-05-20-new-featured-artist-rohi.md

```diff
@@ -0,0 +1,65 @@
+
+We're proud to welcome **Rohi** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/82/header.jpg)
+
+Another blast from the past, [**Rohi**](https://osu.ppy.sh/beatmaps/artists/82) is a Japanese producer most renowned for his Vocaloid works on *Nico Nico Douga*. Tracks like  *Kakuzetsu Thanatos* and *Kodoku Egoism* were all the rage back in the 2013 days of osu!, with groundbreaking mappers like [NatsumeRin](https://osu.ppy.sh/users/151679) taking his sound and using it to lay a foundation of creative work that would influence hundreds of creators going forward.
+
+We've managed to snag **9** historic tracks from **Rohi**'s discography, finally bringing many of these classic maps into the loving embrace of licensed content.
+
+There's still plenty more of **Rohi**'s sound left to discover though, so make sure to give these a listen:
+
+### Kakuzetsu Thanatos
+
+Check out [this classic set](https://osu.ppy.sh/beatmapsets/76396) hosted by [NatsumeRin](https://osu.ppy.sh/users/151679), and [an osu!mania set](https://osu.ppy.sh/beatmapsets/357060) hosted by [_underjoy](https://osu.ppy.sh/users/2235750)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/82/previews/2165.mp3" type="audio/mpeg">
+</audio>
+
+### Kodoku Egoism
+
+Check out [this classic set](https://osu.ppy.sh/beatmapsets/58737) hosted by [NatsumeRin](https://osu.ppy.sh/users/151679)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/82/previews/2167.mp3" type="audio/mpeg">
+</audio>
+
+### Kanata ni Mau wa Sakura no Shirabe
+
+Check out [this osu! and osu!taiko set](https://osu.ppy.sh/beatmapsets/93555) hosted by [NatsumeRin](https://osu.ppy.sh/users/151679), and [an osu!taiko set](https://osu.ppy.sh/beatmapsets/111635) by [mintong89](https://osu.ppy.sh/users/966210)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/82/previews/2166.mp3" type="audio/mpeg">
+</audio>
+
+### Ichiru no Nozomi yo, Ano Tsuki e Hibike
+
+Check out [this classic set](https://osu.ppy.sh/beatmapsets/95148) hosted by [pieguy1379](https://osu.ppy.sh/users/107485)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/82/previews/2164.mp3" type="audio/mpeg">
+</audio>
+
+### Artifact
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/82/previews/2162.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+All of this plus a bunch more are available right now over at [**Rohi**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/82), so if any of this stuff interests you, check out his profile and take a stab at mapping some! Everything is pre-timed and ready to go, so what're you waiting for?
+
+Tweeter @stacker_joe quite literally stacked the deck this week and was the ONLY person to correctly suss out **Rohi** as this week's artist! Not only that, but they correctly deciphered the hint as well, translating 'kakuzetsu' into isolation, and 'thanatos' into death, and bringing it all back together. Nice!
```

Comment:
twitter users drewdeowo, FKnea, ImMyyrh also all guessed it correctly, with drewdeowo and FKnea doing so before stackerjoe also cracked the code. Paragraph could be updated to reflect this?

reference -> https://twitter.com/search?q=rohi%20(to%3Aosugame)&src=typed_query

Reply:
did not know you could search twitter like that, handy

also amazing that exactly 0 of these showed up in my own twitter view